### PR TITLE
fix(readme): Update readme content

### PR DESCRIPTION
**Pull request by AndrePanin**
Updated readme

### CODE REVIEW

**Review by shamilsan — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -5,12 +5,12 @@
   </a>
 </p>
 
-<h4 align="center">
-Gear is a Substrate-based smart-contract platform allowing anyone to run dApp in a few minutes.
-</h4>
+<h3 align="center">
+Gear Protocol is a Substrate-based platform for developers allowing anyone to run dApp in a few minutes.
```
Comment (breathx) [inline README.md:9]: ".. protocol is .. platform" sounds weird

File: README.md [hunk 2]
```diff
@@ -180,7 +176,7 @@ Here are some features in progress or planned: https://github.com/gear-tech/gear
 
 # License
 
-Gear is licensed under [GPL v3.0 with a classpath linking exception](LICENSE).
+Gear Protocol is licensed under [GPL v3.0 with a classpath linking exception](LICENSE).
```
Comment (breathx) [inline README.md:179]: and Vara Network maybe?

File: README.md [hunk 3]
```diff
@@ -86,88 +86,84 @@ Gear node can run in a single Dev Net mode or you can create a Multi-Node local
...
 Modern blockchains solve many issues of the older blockchain networks, such as:
  - Lack of scalability, low transaction speed, high transaction costs
  - Domain-specific development language (high barrier to entry)
  - Complex and inefficient native consensus protocols
  - Absence of intercommunication tools
 
-But still have room for improvements due to:
- - Fixated, rigid native consensus protocols
+But the room for improvements remains related to:
+ - Optimization of the user experience when using web3 applications
```
Comment (breathx) [inline README.md:157]:
```suggestion
 - Optimisation of the user experience when using Web3 applications
```

**Review by breathx — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (shamilsan) [reply README.md:157]: I prefer `Optimization` though.

https://sapling.ai/usage/optimizing-vs-optimising

File: README.md (hunk 2 above)
Comment (shamilsan) [reply README.md:179]: How can we license the network? Only source code may be under some license, I guess.

File: README.md (hunk 1 above)
Comment (AndrePanin) [reply README.md]: Gear Protocol is a Substrate-based solution for developers, enabling anyone to run a dApp in just a few minutes.

**Review by breathx — APPROVED** [review]

**Pull request merged**